funcx_endpoint/funcx_endpoint/executors/high_throughput/interchange.py

```diff
@@ -286,6 +296,7 @@ def __init__(self,
                                "--hb_period={heartbeat_period} "
                                "--hb_threshold={heartbeat_threshold} "
                                "--worker_mode={worker_mode} "
+                               "--container_cmd_options='{container_cmd_options}' "
```

Comment:
There is a single `''` around `'{container_cmd_options}'` to ensure space is correctly escaped between cmd options.

Reply:
The command looks like
```
funcx-manager   -c 1.0 --poll 10 --task_url=tcp://127.0.0.1:54942 --result_url=tcp://127.0.0.1:54886 --logdir=./HighThroughputExecutor/worker_logs --block_id={block_id} --hb_period=30 --hb_threshold=120 --worker_mode=singularity_reuse --container_cmd_options='-H /home/zzli' --scheduler_mode=soft --log_max_bytes=268435456 --log_backup_count=1 --worker_type={worker_type}
```